main.go

```diff
@@ -119,6 +120,12 @@ func main() {
 	// Setup the context that's going to be used in controllers and for the manager.
 	ctx := ctrl.SetupSignalHandler()
 
+	setupLog.Info("starting manager", "version", caphversion.Get().String())
+	if err := mgr.Start(ctx); err != nil {
```

Comment:
you have this code already in https://github.com/syself/cluster-api-provider-hetzner/pull/958/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R224 where you can just add the version infos